core/drivers/crypto/caam/hal/common/registers/jr_regs.h

```diff
@@ -0,0 +1,72 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2018-2019 NXP
+ *
+ * Brief   Job Ring Registers.
+ */
+#ifndef __JR_REGS_H__
+#define __JR_REGS_H__
+
+#include <util.h>
+
+/* Job Ring Block Register Size */
+#define JRx_BLOCK_SIZE		CFG_JR_BLOCK_SIZE
```

Comment:
Defines are supposed to be all upper case, I don't think `JRX_BLOCK_SIZE` is less readable.

Reply:
ok